rubric_data/views.py

```diff
@@ -102,7 +105,8 @@ def denormalize(data):
 
 def get_rating(criterion_id, score, criteria_lookup):
     criterion_info = criteria_lookup[criterion_id]
-    for rating in sort(criterion_info, key=lambda x: x['ratings'], reverse=True):
+    criterion_info['ratings'].sort(reverse=False, key=lambda x: x['points'])
```

Comment:
Yes, I think it does because the points awarded can fall within a range between ratings. So the way the the rating description is assigned (thanks @jaguillette for working this out) is by "walking up" the ratings ladder and assigning the first rating that fits the expression `if value <= another_value` like so:
```
for rating in criterion_info['ratings']:
        if score <= rating['points']:
            return rating['description']
``` 
So, if the ratings are sorted from greatest to least, then all scores will receive the highest rating description. If the ratings are sorted from least to greatest, then the above block should walk up them and assign descriptions accordingly.
I should contribute some tests for the `get_rating` function. That will probably happen on the next rubric_vis story.